frontend/src/components/FormEnter/FormEnter.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+
+import '../../../../styles/FormEnter.css';
+
+function FormEnter() {
+	const [activeRegistration, setActiveRegistration] = useState(true);
+	const [activeEnter, setActiveEnter] = useState(false);
+	const [activeRecovery, setActiveRecovery] = useState(false);
+	const [activeTitle, setActiveTitle] = useState(false);
+
+	const handleClickEnter = () => {
+		setActiveRegistration(false);
+		setActiveEnter(true);
+		setActiveRecovery(false);
+		setActiveTitle(false);
+	}
+
+	const handleClickRegistration = () => {
+		setActiveRegistration(true);
+		setActiveEnter(false);
+		setActiveRecovery(false);
+		setActiveTitle(false);
+	}
+
+	const handleClickRecovery = () => {
+		setActiveRegistration(false);
+		setActiveEnter(false);
+		setActiveRecovery(true);
+		setActiveTitle(true);
+	}
+
+	return (
+		<section className="form-enter">
+			<form action="" className="form-enter__container">
+				<div className="form-enter__content">
+					<div className={`form-enter__wrapper-title ${activeTitle ? 'visibility' : ''}`}>
+						<p
+							onClick={handleClickRegistration}
+							className={`form-enter__title ${activeRegistration ? 'form-enter__title_active' : ''}`}>
+							Регистрация
+						</p>
+						<p
+							onClick={handleClickEnter}
+							className={`form-enter__title ${activeEnter ? 'form-enter__title_active' : ''}`}>
+							Вход
+						</p>
+					</div>
+					<p className={`form-enter__title ${activeRecovery ? '' : 'visibility'}`}>Забыли пароль?</p>
+					<div className="form-enter__input-container">
+						<Input
+							inputType={'email'}
+							inputClassName={'input'}
+							inputPlaceHolder={"Email"}
+						/>
+
+						<div className={`form-enter__wrapper-input ${activeRecovery ? 'visibility' : ''}`}>
+
+							<Input
+								inputType={'password'}
+								inputClassName={'input input_type_password'}
+								inputPlaceHolder={"Введите пароль"}
+							/>
+						</div>
+						{/* <div className={`form-enter__wrapper-input ${activeRecovery ? 'visibility' : ''}`}>
+							<Input
+								inputType={'password'}
+								inputClassName={'input input_type_password'}
+								inputPlaceHolder={"Повторите пароль"}
+							/>
+						</div> */}
```

Comment:
Предлагаю использовать не стили для отображения компонентов, а условный рендеринг. В данном случае еще важно передать инпутам уникальный проп key, для того, чтобы при переключении между вкладками стирались  ранее набранные почта и пароль

```suggestion
<Input key={`${formView}-email`} inputClassName="input" inputType="email" inputPlaceHolder="Email" />
	{formView !== formState.recovery && (
		<div className='form-enter__wrapper-input'>
			<Input
				key={`${formView}-password`}
				inputType="password"
				inputClassName="input input_type_password"
				inputPlaceHolder="Введите пароль"
			/>
		</div>
	)}
```